app/views/scan_analytics/_index.html.haml

```diff
@@ -0,0 +1,67 @@
+:ruby
+  scan_data = @scan_data.present? ? @scan_data : scan_data
+  scan_data = JSON.parse(scan_data.data.to_json, object_class: OpenStruct)
+  @cwes = scan_data.cwe
+  version = scan_data.analysis_metrics.version.present? ? scan_data.analysis_metrics.version : nil
+  last_analysed = scan_data.analysis_metrics.build_date
+  loc = scan_data.analysis_metrics.loc
+  defect_density = scan_data.analysis_metrics.defect_density&.score
+  total_defects = scan_data.analysis_metrics.total_count
+  outstanding_count = scan_data.analysis_metrics.outstanding_count
+  fixed_count = scan_data.analysis_metrics.fixed_count
+#scan_data
+  .row.row-eq-height.project_row
+    %h4#scan_analytic.text-left Static Analysis
+  - if @analytics.size > 1
+    .row.row-eq-height.project_row
+      .pull-left#scan_analytic#scan_dropdown
+        = select_tag :code_set_id, options_for_select(@analytics.collect{ |u| [u.code_set.code_location.url, u.code_set_id] }, params[:code_set_id]), class: 'chzn-select value-select', data: { remote: true, url: project_scan_analytics_path(@project), method: 'get' }
```

Comment:
This line is too long. Please split it over multiple lines.

Reply:
It's written in multiple lines in the editor. I don't know why it's displaying here as a single line.
<img width="1512" alt="Screenshot 2022-11-02 at 10 09 33 PM" src="https://user-images.githubusercontent.com/101638919/199549341-cb8a09ea-7932-4175-b872-900fa6412267.png">

Reply:
Seems like your editor auto `wraps` it for you. There is a setting for it. Consider disabling the setting.